index.html

```diff
@@ -18,8 +18,8 @@
     <meta charset="UTF-8" />
```

Comment:
- Provide a Pull Request description about the features implemented on this branch.

Reply:
Hi @carlos-ssh ... I've added the description.